howso/engine/trainee.py

```diff
@@ -524,12 +529,13 @@ def copy(
             params["project_id"] = project_id
 
         if isinstance(self.client, AbstractHowsoClient):
-            copy = self.client.copy_trainee(**params)
+            trainee_data = self.client.copy_trainee(**params)
+            copy = Trainee(**trainee_data)
```

Comment:
Would be safer to have a from_dict() method I think, that uses the attribute_map. That way any additional data that is not tracked by these classes would not raise an error since the kwarg doesnt exist. And it would be preferable to having **kwargs in the `__init__` method as well.

Reply:
Same would go for all the engine.* classes

Reply:
Addressed partially in: c28e6de8dded39f04ba3206735fe885f931d8d2e

I couldn't find similar copy functions in any of the engine.* classes. Were there specific ones I missed in my search? Project, Session and Client don't seem to have a similar copy function.

Reply:
This was in reference to the removal of `from_openapi`, should be using `from_dict` instead.